radio/src/targets/common/arm/stm32/bluetooth_driver.cpp

```diff
@@ -24,7 +24,9 @@
 Fifo<uint8_t, BT_TX_FIFO_SIZE> btTxFifo;
 Fifo<uint8_t, BT_RX_FIFO_SIZE> btRxFifo;
 
-#if defined(PCBX7) || defined(PCBXLITE)
+#if defined(PCBX7ACCESS)
```

Comment:
the code start to be too much complex here, I propose we introduce:
- BLUETOOTH=YES
- BLUETOOTH=NO
- BLUETOOTH=PROBE

Reply:
I will it myself later